src/js/controllers/imageCtrl.js

```diff
@@ -0,0 +1,27 @@
+(function (angular) {
+    angular
+        .module('flickrPOC')
+        .controller('imagesCtrl', imagesCtrl);
+
+    imagesCtrl.$inject = ['images', '$log'];
+
+    function imagesCtrl(images, $log) {
+        var vm = this;// jshint ignore:line
+
+        function activate(){
+
+            images.loadPics()
+                .then(function (res) {
+                    vm.images = res;
+                    return vm.images;
```

Comment:
Is this statement needed to pass your tests? I don't think it should be. In this case, we're not really looking to return a value. Once we set the value to `vm.images`, Angular's data-binding will update any uses. If this is required to pass tests, let's take a look together.